user-service/src/main/java/sjchat/users/UserServiceServer.java

```diff
@@ -48,13 +52,22 @@ private void blockUntilShutdown() throws InterruptedException {
 
   static class UserService extends UserServiceGrpc.UserServiceImplBase {
 
+    UserDao dao;
+    public UserService(){
+      super();
+      dao = new UserDaoImpl();
+    }
+
     @Override
     public void createUser(UserDataRequest req, StreamObserver<UserResponse> responseObserver) {
       Random random = new Random();
 
+      UserEntity entity = new UserEntity(Math.abs(random.nextInt(100)), req.getUsername());
```

Comment:
Why is the user id set to a random number here? Can't it be generated by the database, maybe as an UUID instead?